### PR TITLE
Added "TotalCode" Ecommerce Platform

### PR DESCRIPTION
**Pull request by diego-reinoso**
TotalCode is an e-commerce and omnichannel platform

### CODE REVIEW

**Comment by diego-reinoso** [comment]
Sample Sites:
http://www.easy.com.co/
http://www.floresymas.com/
http://www.homesentry.co/

File: src/apps.json [hunk 1]
```diff
@@ -8144,6 +8144,16 @@
 			"icon": "TornadoServer.png",
 			"website": "http://tornadoweb.org"
 		},
+		"TotalCode": {
+			"cats": [
+				"6"
+			],
+			"headers": {
+				"X-Powered-By": "TotalCode"
```
Comment (jvoisin) [inline src/apps.json]: Please use `"X-Powered-By": "^TotalCode$"` instead :)

Comment (diego-reinoso) [inline src/apps.json]: Thanks for your input! Requested anchors have been added.

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**